util/ghostping-v2.go.tmpl

```diff
@@ -0,0 +1,47 @@
+{{/*
+    Ghost-Ping Detector CC v2
+
+    Recommended Trigger Type: Regex
+    Recommended Trigger: .*
+    Optional Trigger: <@!?\d{17,19}>
+    
+    Credits:
+    Devonte <https://github.com/devnote-dev>
+*/}}
+
+{{/* THINGS TO CHANGE */}}
+
+{{$CHECK := false}} {{/* Change to "true" for double exec-check */}}
+
+{{$LOG := false}} {{/* Change to "true" to create message logs */}}
+
+{{/* ACTUAL CODE - DO NOT TOUCH */}}
+
+{{if .ExecData}}
+    {{$mentions := ""}}{{$ping := false}}
+    {{if ($m := getMessage nil .ExecData.message)}}
+    {{if $m.Mentions}}{{else}}{{$ping = true}}{{end}}
+    {{else}}{{$ping = true}}{{end}}
+    {{if $ping}}
+    {{range .ExecData.mentions}}{{$mentions = print $mentions "<@" .ID ">, "}}{{end}}
```

Comment:
Same issue as V1: This will always have the message end with a `,`
![image](https://user-images.githubusercontent.com/3849946/114313746-20d98980-9ace-11eb-8678-b61b66832b42.png)